libs/game/tilemap.ts

```diff
@@ -98,6 +96,10 @@ namespace tiles {
             }
         }
 
+        enabled(): boolean {
```

Comment:
you can make this a property ``get enabled(): boolean``

Reply:
For some reason I blanked on that, thanks